extensions/rich_text_components/Collapsible/Collapsible.html

```diff
@@ -1,4 +1,10 @@
 <script type="text/ng-template" id="richTextComponent/Collapsible">
+  <style>
+    .conversation-skin-tutor-card-top-content.protractor-test-conversation-content {
```

Comment:
Actually, I just realized -- why is this here? It doesn't seem to have anything to do with the collapsible. Should it go in player/ instead?

Also, note that the .protractor-test-* classes are used only for e2e tests, and shouldn't be used in CSS. You may need to omit the reference to that class, and/or add a new class if needed (though I'm guessing you could just augment the existing CSS rule for .conversation-skin-tutor-card-top-content?)